iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -2420,11 +2464,36 @@ private String timestampAfterSnapshot(Table table, int snapshotPosition) {
     return simpleDateFormat.format(new Date(time));
   }
 
-  private void checkColStat(String tableName, String colName) {
+  private void checkColStat(String tableName, String colName, boolean accurate) {
     List<Object[]> rows = shell.executeStatement("DESCRIBE " + tableName + " " + colName);
 
     Assert.assertEquals(2, rows.size());
     Assert.assertEquals(StatsSetupConst.COLUMN_STATS_ACCURATE, rows.get(1)[0]);
+    Assert.assertEquals(accurate, !rows.get(1)[1].toString().matches("\\{\\}\\s*"));
```

Comment:
Can you add a small comment explaining the regex here?

Reply:
Added